platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py

```diff
@@ -519,6 +519,15 @@ def monitor_asic_themal_zone(cls):
         else:
             cls.expect_cooling_state = None
 
+    @classmethod
+    def start_cpu_thermal_control(cls, chassis):
+        platform_name = DeviceDataManager.get_platform_name()
+        if platform_name != 'x86_64-nvidia_sn4800-r0':
```

Comment:
should we add it as a new capability list and when there is new system that supports it it will be in such list and not in different places in the code as if statement?

Reply:
unused code, will remove.